bokeh/core/enums.py

```diff
@@ -179,3 +179,5 @@ def enumeration(*values, **kwargs):
 #: Specify sorting directions
 SortDirection = enumeration("ascending", "descending")
 
+#: Responsive types
+Responsive = enumeration("box", "width_ar", "height_ar", "box_ar", "fixed")
```

Comment:
To me `width_aspect` sounds like width-aspect. Something like `width_fixed_aspect_ratio` would be clear but way too long. `width_far` or `width_ar` is a bit cryptic imo. 

What about just 'width', 'height', 'both' for the fixed ar ones? Then 'fixed' can stay, and for what is now 'box' we can use 'free' or 'full', or keep it 'box'.